cartographer_ros/cartographer_ros/CMakeLists.txt

```diff
@@ -66,3 +66,14 @@ install(TARGETS cartographer_occupancy_grid_node
   LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
+
+google_binary(cartographer_rosbag_wtf
+  SRCS
+    rosbag_wtf_main.cc
+)
+
+install(TARGETS cartographer_rosbag_wtf
```

Comment:
Maybe a different name (for binary and source) sounds friendlier, but I leave the decision to you.

Reply:
I chose the name for symmetry with `roswtf`, but I am not a fan of the name myself. After offline discussion the benefit of the symmetry of naming is less important than the unpleasant connotations of the vulgar name. I therefore renamed the tool.